test/v1_integration.py

```diff
@@ -360,14 +360,14 @@ def test_issuer():
     store_ctx.verify_certificate()
 
 def test_ocsp():
-    cert_file_pem = os.path.join(tempdir, "cert.pem")
-    auth_and_issue([random_domain()], cert_output=cert_file_pem)
-
-    ee_ocsp_url = "http://localhost:4002"
+    cert_file = tempfile.NamedTemporaryFile(
+        dir=tempdir, suffix='.test_ocsp.pem', delete=False)
+    cert_file.close()
+    auth_and_issue([random_domain()], cert_output=cert_file.name)
```

Comment:
I notice this pattern recurs a lot: Create a tempfile, close it, pass the filename to auth_and_issue. Since chisel + auth_and_issue are part of our integration tests, we should feel free to tweak them for our needs. Proposal: make the cert_output param take a file handle rather than a file name. That way we can skip the cert_file.close() step. Note that this would need two changes: one for chisel.py and one for chisel2.py.

Also the invocation of tempfile.NamedTemporaryFile is wordy and a bit repetitive. How about creating a helper function that calls it with the common parameters here?

Reply:
I've created a new helper function that cleans things up a lot. Although I said I was going to make it a context manager, the different ways these files get used makes it a little cleaner for it to just be a plain function (and to rely on the @atexit registered cleanup to clean up all the temp files).